packages/engine/src/proto.rs

```diff
@@ -229,6 +230,8 @@ pub struct SimpleExperimentConfig {
     /// Number of steps each run should go for
     #[serde(rename = "numSteps")]
     pub num_steps: usize,
+    /// Maximum amount of simulations that can be ran in parallel - None is unlimited
+    pub max_sims_in_parallel: Option<usize>,
```

Comment:
Do we want to document this somewhere?

Reply:
I thought we hadn't documented `experiments.json` in the README yet but it turns out we have. Will add

Reply:
@Alfred-Mountfield @TimDiekmann We have a [running Google Doc](https://docs.google.com/document/d/14lsNI-hxiTidc1QmkFqKWGt5KFoWlUnPbAPIaYmV2uA/edit#heading=h.puo3e05fkth) for stuff we'll feature on the forthcoming `hash.dev` site that will host documentation for our self-hosted projects. I've added a section entitled 'hEngine Documentation' at the end.